packages/frontend/src/components/common/modal/ChartCreateModal/SaveToSpaceOrDashboard.tsx

```diff
@@ -338,24 +401,38 @@ export const SaveToSpaceOrDashboard: FC<Props> = ({
                     padding: theme.spacing.md,
                 })}
             >
-                <Button onClick={onClose} variant="outline">
-                    Cancel
-                </Button>
-
-                <Button
-                    type="submit"
-                    loading={isSavingChart || isSavingSpace}
-                    disabled={
-                        !form.values.name ||
-                        (!form.values.newSpaceName &&
-                            saveDestination === SaveDestination.Space &&
-                            !form.values.spaceUuid) ||
-                        (!form.values.dashboardUuid &&
-                            saveDestination === SaveDestination.Dashboard)
-                    }
-                >
-                    Save
-                </Button>
+                {currentStep === ModalStep.InitialInfo ? (
+                    <>
+                        <Button onClick={onClose} variant="outline">
+                            Cancel
+                        </Button>
+                        <Button
+                            onClick={(
+                                e: React.MouseEvent<HTMLButtonElement>,
```

Comment:
huh, strange. I thought events were typed in manine

Reply:
yea that came from react 19 upgrade - we might not have that issue with mantine-v7 🤔